examples/dgmulti_2d/elixir_euler_shockcapturing_curved.jl

```diff
@@ -0,0 +1,56 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations2D(1.4)
+
+initial_condition = initial_condition_weak_blast_wave
```

Comment:
Did you try running a bit more demanding example such as the IC in `examples/tree_2d_dgsem/elixir_euler_blast_wave.jl`?

Reply:
I can try

Reply:
I think I need to finish https://github.com/trixi-framework/Trixi.jl/pull/1299/ to run the stronger blast wave - the entropy projection is producing `NaN`s.

Reply:
I added this to https://github.com/trixi-framework/Trixi.jl/issues/1470

Reply:
:+1: